src/parameterizations/vertical/MOM_internal_tide_input.F90

```diff
@@ -389,10 +404,17 @@ subroutine int_tide_input_init(Time, G, GV, US, param_file, diag, CS, itide)
                  "If true, apply an arbitrary generation site for internal tide testing", &
                  default=.false.)
   if (CS%int_tide_source_test)then
+    call get_param(param_file, mdl, "INTERNAL_TIDE_USE_GLOB_IJ", CS%int_tide_use_glob_ij, &
+                 "Use global IJ for interal tide generation source test", default=.false.)
     call get_param(param_file, mdl, "INTERNAL_TIDE_SOURCE_X", CS%int_tide_source_x, &
```

Comment:
Depending on the value of INTERNAL_TIDE_USE_GLOB_IJ, only one for the two sets of parameters INTERNAL_TIDE_SOURCE_[XY] or INTERNAL_TIDE_SOURCE_[IJ] will get used. I think that maybe it would make sense to (1) only log the pair that will be used (i.e., set `do_not_log=CS%int_tide_use_glob_ij` on lines 410 and 412 and `do_not_log=.not.CS%int_tide_use_glob_ij` on lines 414 and 416 and (2) add a useful error message if one of the parameters that will not get used is given a non-default value.

Reply:
added